### ISSUE TITLE
Squiz.Commenting.FunctionComment.ThrowsNoFullStop false positive when one line @throws

### ISSUE THREAD

**Issue by vaclavvanik** [squizlabs/PHP_CodeSniffer#3503]
**Describe the bug**
`Squiz.Commenting.FunctionComment` sniff throws `ThrowsNoFullStop` on one line php doc comment with @throws annotation.

**Code sample**
```php
<?php

declare(strict_types=1);

namespace My;

use Exception;

/** @throws Exception if any other error occurs. */
function foo(string $string): string
{
    return $string;
}
```

**To reproduce**
1. Create a file called `test.php` with the code sample above...
2. Run `phpcs test.php ...`
3. See error message displayed
```
FOUND 1 ERROR AFFECTING 1 LINE
------------------------------------------------------------------------------------------------------------------
  9 | ERROR   | @throws tag comment must end with a full stop
    |         | (Squiz.Commenting.FunctionComment.ThrowsNoFullStop)

```

**Expected behavior**
No error :-)

**Versions (please complete the following information):**
 - OS: Ubuntu 18.04
 - PHP: 8.0
 - PHPCS: 3.6.1
 - Standard: Squiz

**Comment by gsherwood** [comment]
Thanks for the report. The sniff didn't account for the whitespace after the comment, which only occurs when the comment is closed on the same line. Fixed now, and will be in 3.7.0.
